src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -24,12 +24,19 @@ public static class ScriptConstants
         public const string TracePropertyEventDetailsKey = "MS_EventDetails";
         public const string TracePropertyIsUserTraceKey = "MS_IsUserTrace";
         public const string TracePropertyIsSystemTraceKey = "MS_IsSystemTrace";
+        public const string TracePropertyFunctionInvocationIdKey = "MS_FunctionInvocationId";
+        public const string TracePropertyHostIdKey = "MS_HostId";
+        public const string TracePropertyScriptHostInstanceIdKey = "MS_ScriptHostInstanceId";
 
         public const string TraceSourceSecretManagement = "SecretManagement";
         public const string TraceSourceHostAdmin = "HostAdmin";
         public const string TraceSourceFileWatcher = "FileWatcher";
         public const string TraceSourceSwagger = "SwaggerDocumentation";
         public const string TraceSourceHttpHandler = "HttpRequestTraceHandler";
+        public const string TraceSourceScriptHost = "Script.Host";
```

Comment:
@fabiocav - Any ideas on what should be the "source" column value in our logs? Do we want to see similar to what [webjobs sdk](https://github.com/Azure/azure-webjobs-sdk/blob/663a508e8a851629c26a51e7de3af36629dfd120/src/Microsoft.Azure.WebJobs.Host/TraceSource.cs) has like "Script.Host" or should it be the class name like [this](https://github.com/Azure/azure-functions-host/blob/v1.x/src/WebJobs.Script/Description/DotNet/DotNetFunctionInvoker.cs#L196)